src/game.cpp

```diff
@@ -12222,6 +12222,13 @@ void game::start_calendar()
                               + get_option<int>( "SEASON_LENGTH" ) * 1_days * scen->initial_season()
                               + 4 * get_option<int>( "SEASON_LENGTH" ) * 1_days * ( scen->initial_year() - 1 )
                               + 1_days * get_option<int>( "SPAWN_DELAY" );
+    if( calendar::start_of_game < calendar::start_of_cataclysm ) {
+        // Hotfix to prevent game start  from occuring before the cataclysm.
+        // Should be replaced with full refactor of the start date
+        calendar::start_of_game = calendar::start_of_cataclysm
+                                  + 1_hours * scen->initial_hour()
+                                  + 1_days * get_option<int>( "SPAWN_DELAY" );
+    }
```

Comment:
In other words, you are suggesting:
```suggestion
                              + 4 * get_option<int>( "SEASON_LENGTH" ) * 1_days * ( scen->initial_year() - 1 );
    if( calendar::start_of_game < calendar::start_of_cataclysm ) {
        // Hotfix to prevent game start  from occuring before the cataclysm.
        // Should be replaced with full refactor of the start date
        calendar::start_of_game = calendar::start_of_cataclysm
                                  + 1_hours * scen->initial_hour();
    }
    calendar::start_of_game += 1_days * get_option<int>( "SPAWN_DELAY" );
```

Reply:
Indeed.